assets/javascript/game.js

```diff
@@ -1,76 +1,151 @@
+
   outsourcedWork();
   updateMoney();
+  setupModalTriggers();
+  updateDeveloper(mainDeveloper);
 };
 
 function outsourcedWork() {
   developers.forEach(developer => {
-    if(developer.level > 0) {
+    if(developer.level > 0 && developer !== mainDeveloper) {
       writtenLines += developer.performance();
     }
   });
+
   updatePerformance();
   setTimeout(outsourcedWork, 1000);
 }
 
 function updateMoney() {
-  moneyElement.innerHTML = '$'+money
+  moneyElement.innerHTML = '$' + money;
 }
 
 function updatePerformance() {
   writtenLinesElement.innerHTML = "You have written " + writtenLines + " lines of code!";
+
   if (writtenLines >= project.linesOfCode) {
     writtenLines = 0;
     givePayment(project.payment);
     stage++;
     project = new Project;
-    stageElement.innerHTML = "You are in the stage: "+stage;
+    stageElement.innerHTML = "You are in the stage: " + stage;
   }
 }
 
 function givePayment(value) {
   money += value;
+
   updateMoney();
 }
 
-function improveSelf() {
-  if(money >= (Math.pow(click_value, 2))) {
-    money = money - (Math.pow(click_value, 2));
+function improveDeveloper(event) {
+  let developerId = event.target.parentNode.parentNode.id;
+  let developer = developers.find(developer => developer.id === developerId);
+  let price = developer.price();
+
+  if(money >= price) {
+    money = money - price;
+    developer.levelUp(1);
+
+    if(developer.level === 1) {
+      employees.innerHTML += '<img alt="' + developer.name + '" class="developer" src="assets/images/employees/' + developer.id + '.png">';
+    }
+
+    updateDeveloper(developer);
     updateMoney();
-    click_value++;
   }
 }
 
-function hireDeveloper() {
-  let developer = developers[helper_count];
-  if(money >= developer.price) {
-    money = money - developer.price;
-    updateMoney();
-    helper_count ++;
-    addNewDeveloper(developer);
+function updateDeveloper(developer) {
+  let developerElement = document.getElementById(developer.id);
+  let developerPosition = developers.indexOf(developer);
+  let developerAttributes = developerElement.childNodes[3];
+
+  developerElement.childNodes[1].classList = 'developer__icon';
+  developerAttributes.childNodes[1].classList = 'developer__attribute';
+  developerAttributes.childNodes[3].innerHTML = 'Level: ' + developer.level;
+  developerAttributes.childNodes[3].classList = 'developer__attribute';
+  developerAttributes.childNodes[5].innerHTML = 'Lines/Click: ' + developer.performance();
+  developerAttributes.childNodes[5].classList = 'developer__attribute';
+  developerAttributes.childNodes[7].innerHTML = 'Price: ' + developer.price() + '$';
+  developerAttributes.childNodes[7].classList = 'developer__attribute';
+  developerAttributes.childNodes[9].innerHTML = 'Improve skills';
+
+  if(developersElement.lastElementChild === developerElement && developerPosition < developers.length-1) {
+    addNewMenuDeveloper(developers[developerPosition+1]);
   }
 }
 
 function click() {
```

Comment:
What do u think about rename this function? something more intuitive, since we have another click handlers